core/arch/x86/decode_table.c

```diff
@@ -1976,6 +1976,24 @@ const instr_info_t * const op_instr[] =
 #define vsiby    REQUIRES_VSIB_YMM
 #define vsibz    REQUIRES_VSIB_ZMM
 
+/* AVX-512 tupletype attributes. They are moved into the upper half
+ * of the instr_info_t flags.
+ */
+#define ttnone (TUPLE_TYPE_NONE * 65536)
```

Comment:
I'll change this to << 16, just in case the compiler doesn't strength reduce (every compiler should).

Reply:
Yes please, not just for the compiler, for human readers: these are bit fields, so multiplication is confusing.

Reply:
Done.